### PR TITLE
Updated ispc.rst with description of float16 functions

### PR DESCRIPTION
**Pull request by aneshlya**

### CODE REVIEW

File: docs/ispc.rst [hunk 1]
```diff
@@ -756,10 +756,23 @@ on which you're running ``ispc`` is used to determine the target CPU.
...
+For example, ``--target=avx2-i32x16`` specifies a target with the AVX2 instruction set,
+a mask size of 32 bits, and a gang size of 16.  For targets with hardware masking support,
+which are AVX512 and GPU targets, the target string is of the form
+``[ISA]-x[gang size]``.  For example, ``--target=xehpg-x16`` specifies Intel XeHPG
+as a target ISA and defines a gang size of 16.
+
+By default, the target instruction set is chosen based on the most capable
+one supported by the system on which you're running ``ispc``.  In this case a warning
+will be issued noting the target used for compilation.  It is recommended to
+always used ``--target`` switch to explicitly specify the target.
```
Comment (aneshlya) [inline docs/ispc.rst:770]: to always used -> to always use

File: docs/ispc.rst [hunk 2]
```diff
@@ -496,9 +496,23 @@ code.
...
+avx512skx-x32, avx512skx-x64, and avx512knl-x16.
+
+Standard library gained full support for ``float16`` type.  Note that it is
+fully supported only on the targets with native hardware support.
+On the other targets emulation is still not guaranteed, but may work in some cases.
+
+The compiler gained support for ``-E`` switch for running preprocessor only,
+which is similar to the switch of C/C++ compilers.  Also, as a result of bug fix,
+in case of preprocessor error, the compiler will crash now.  It used not to crash and
+produced some output (sometime correct!).  As it was a convenient feature for some
```
Comment (aneshlya) [inline docs/ispc.rst:510]: sometime->sometimes

File: docs/ispc.rst (hunk 1 above)
Comment (dbabokin) [reply docs/ispc.rst]: fixed

File: docs/ispc.rst (hunk 2 above)
Comment (dbabokin) [reply docs/ispc.rst]: fixed

**Pull request merged**